traefik-mixin/config.libsonnet

```
@@ -0,0 +1,26 @@
+{
+  _config+:: {
+    // alerts thresholds
+    traefik_tls_expiry_days_critical: 7,
+    traefik_tls_expiry_days_warning: 14,
+    timeSeriesLabels: '',
+    // Example:
+    // timeSeriesLabels: "component=\"traefik\",environment=\"production\"",
+    // for config alert
+    sumByLabels: 'instance',
+    // for TLS alerts
+    maxByLabels: 'sans',
+    alertLabels: {},
+    // Example:
+    // alertLabels: {
+    //   environment: 'production',
+    //   component: 'traefik',
+    // },
```

```suggestion
    groupLabels: ['job'],
    intanceLabels: ['instance'],
    alertLabels: {},
    // Example:
    // alertLabels: {
    //   environment: 'production',
    //   component: 'traefik',
    // },
```
again, by convention we use to lists, groupLabels, instanceLabels for this case.
instanceLabels - labels that identify specific service instance, most of the time it could be `instance` label.
groupLabels - labels that identify group of service instances. It could be job, namespace, cluster, environment, or any combination of them.

groupLabels or instanceLabels could be used in alerts aggregations, so those labels are kept.
I think it is better to add `environment` label into groupLabels, as its value wouldn't be static but inherited from simliar label from traefik metric.

as of sans, I would hardcode it in the aggregation query itself, as it is always required for annotations templates. Then it could be:
`max by (%(groupLabelsString)s, sans)...`

and
`sum by (%(groupLabelsString)s, sans)...`

something like this.

---

makes sense to me and updated to match convention here! 13e9ae408f233b169540ae4dda10f1f1b1fde817